src/components/ProfilePage/ProfileAvatar.jsx

```diff
@@ -60,62 +60,68 @@ class ProfileAvatar extends React.Component {
   renderMenu() {
     if (!this.props.isEditable) return null;
 
-    if (this.props.isDefault) {
-      return (
-        <Button
-          className="text-white btn-block"
-          color="link"
-          size="sm"
-          onClick={this.onClickUpload}
-        >
-          <FormattedMessage
-            id="profile.profileavatar.upload-button"
-            defaultMessage="Upload Photo"
-            description="Upload photo button"
-          />
-        </Button>
-      );
-    }
-
-    return (
-      <Dropdown
-        isOpen={this.state.dropdownOpen}
-        toggle={this.toggleDropdown}
-      >
-        <DropdownToggle className="text-white btn-block" color="link" size="sm">
-          <FormattedMessage
-            id="profile.profileavatar.change-button"
-            defaultMessage="Change"
-            description="Change photo button"
-          />
-        </DropdownToggle>
-        <DropdownMenu>
-          <DropdownItem onClick={this.onClickUpload}>
+    const menuContent = ((isDefault) => {
```

Comment:
This pattern of creating anonymous functions and passing the props in feels unnecessarily complex to me... It's not unreasonable to just use `this.props.isDefault` directly, and/or to just write these as instance methods.  It feels a bit odd that we have two different ways of writing render methods - one using instance methods and one with these anonymous functions.  

While technically in Javascript the instance methods are not 'private', practically speaking no one outside the component will ever call them nor should, so it feels okay to me to just create helper instance methods.  It keeps the code a bit flatter/keeps each method succinct.